packages/grid/_modules_/grid/models/gridComponentOverridesProp.tsx

```diff
@@ -8,25 +9,29 @@ export interface GridComponentOverridesProp {
   /**
    * Pagination component rendered in the grid footer by default.
    */
-  pagination?: React.ElementType<ComponentProps>;
+  Pagination?: React.ElementType<ComponentProps>;
```

Comment:
This is technically a breaking change, right? How are we handling things like that? Also, these are not specifically listed in the docs, only the interface is.

Reply:
> This is technically a breaking change, right? 

It's a breaking change yes. We have multiple demos using it in the documentation.

> How are we handling things like that?

We mention it in the pull request description and in the changelog.